momento-sdk/build.gradle.kts

```diff
@@ -25,6 +26,9 @@ dependencies {
     // Test dependencies
     testImplementation(libs.junit)
     testImplementation(libs.assertj)
+    testImplementation(libs.slf4j.api)
```

Comment:
oh, okay then I don't think you need to add it as a test dep, it will already be available from the main implementation deps?

should be harmless especially since y'all consolidated the version numbers / artifacts to a single place, so nbd if you have a reason you want to keep it.